### PR TITLE
unit tests for components

### PR DESCRIPTION
**Pull request by Untouchamp**

### CODE REVIEW

File: src/components/Authentification/AuthSection/AuthSection.test.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import firebase from 'firebase/compat';
+import AuthSection from './AuthSection';
+import '@testing-library/jest-dom';
+
+describe('AuthSection component', () => {
+    const signInGoogle = jest.fn();
+    const signOutCurrentUser = jest.fn();
+    const mockUser = {
```
Comment (Gudashev) [inline src/components/Authentification/AuthSection/AuthSection.test.tsx:10]: Remove this mock and skip this test

File: src/components/StylizedElements/StylizedButton/StylizedButton.tsx [hunk 2]
```diff
@@ -1,15 +1,19 @@
 import React from 'react';
-import { StylizedButtonType } from '../ToDoList/types';
+import { StylizedButtonType } from '../../ToDoList/types';
 
 function StylizedButton(props: StylizedButtonType): JSX.Element {
     const { clickHandler, buttonDescription } = props;
 
+    const onClickHandler = () => {
```
Comment (Gudashev) [inline src/components/StylizedElements/StylizedButton/StylizedButton.tsx:7]: You don't this function. You can use the callback

File: src/components/ToDoList/Todo/Todo.test.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import Todo from './Todo';
+import '@testing-library/jest-dom';
+
+const descriptionMock = 'Dummy';
+
+describe('Todo component', () => {
+    test('renders correctly', () => {
+        const task = {
```
Comment (Gudashev) [inline src/components/ToDoList/Todo/Todo.test.tsx:10]: You use it several times. Take it outside

File: src/components/ToDoList/TodoWrapper/TodoWrapper.tsx [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
     const [
         todos,
         addTask,
         deleteTask,
         updateTask,
-        dispatch,
-    ]: CustomTypeHooksType = useCustomTaskHooks();
+    ] = useCustomTaskHooks();
+
+    console.log(todos);
```
Comment (Gudashev) [inline src/components/ToDoList/TodoWrapper/TodoWrapper.tsx:14]: remove

**Review by Gudashev — APPROVED** [review]

File: src/components/ToDoList/TodoWrapper/TodoWrapper.tsx (hunk 4 above)
Comment (Untouchamp) [reply src/components/ToDoList/TodoWrapper/TodoWrapper.tsx]: image

**Pull request merged**